### PR TITLE
Add .editorconfig

### PR DESCRIPTION
**Pull request by timkurvers**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (sindrenm) [inline .editorconfig:12]: Why do we want trailing whitespaces in Markdown files?

Comment (timkurvers) [reply .editorconfig:12]: From the [Markdown documentation](https://daringfireball.net/projects/markdown/syntax):

> When you do want to insert a &lt;br /&gt; break tag using Markdown, you end a line with two or more spaces, then type return.

Comment (sindrenm) [reply .editorconfig:12]: That's just completely silly. :neutral_face:

**Comment by sindrenm** [comment]
image

**Pull request merged**